packages/io/x3d-serializer/tests/serialize.test.js

```diff
@@ -57,9 +59,9 @@ const expected3 = `<?xml version="1.0" encoding="UTF-8"?>
       </IndexedTriangleSet>
     </Shape>
     <Shape>
-      <IndexedTriangleSet ccw=\"true\" colorPerVertex=\"false\" solid=\"false\" index=\"0 1 2 0 3 1 4 5 6 4 7 5 0 6 3 0 4 6 2 5 7 2 1 5 0 7 4 0 2 7 3 5 1 3 6 5\">
-        <Coordinate point=\"4 4 4 4 6 6 4 6 4 4 4 6 6 4 4 6 6 6 6 4 6 6 6 4\"/>
-        <Color color=\"0.5 1 0.5 0.5 1 0.5 0.5 1 0.5 0.5 1 0.5 0.5 1 0.5 0.5 1 0.5 0.5 1 0.5 0.5 1 0.5 0.5 1 0.5 0.5 1 0.5 0.5 1 0.5 0.5 1 0.5\"/>
+      <IndexedTriangleSet ccw="true" colorPerVertex="false" solid="false" index="0 1 2 0 3 1 4 5 6 4 7 5 0 6 3 0 4 6 2 5 7 2 1 5 0 7 4 0 2 7 3 5 1 3 6 5">
```

Comment:
the newer version seems more logical and does not have the weird backlashes, but is it correct X3D ?

Reply:
Ahh... These changes from from eslint, which suggested to remove those backslashes.

Correct X3D? If there's an official 'validator' there we could make sure the serialized designs are correct X3D.

Reply:
No clue, I barely ever heard of x3d at all before the serializer to be honest :-) 
Seems like cleaner XML in the V2 version though so all good :-)